src/modules/storage/LedgerStorage.ts

```diff
@@ -867,10 +867,40 @@ export class LedgerStorage extends Storages
             `SELECT
                 TX.address,
                 TX.block_height as height,
+                STRFTIME('%s', '2020-01-01 00:00:00') * 1000 + TX.block_height * 10 * 60000 as block_time,
```

Comment:
This is seen as a temporary measure. It will change when time stamps are added to the block table in the future.
Is it correct?

Reply:
Yes

Reply:
I added Todo comment